### ISSUE TITLE
Wrong indentation after namespaced pipes

### ISSUE THREAD

**Issue by vspinu** [emacs-ess/ESS#851]
This one is a true corner case as it occurs only after the namespaced invocation `xxx` inside a `testthat` block:

```R
test_that("", {

  aa <-
    xxx::fn1() %>%
    fn2(bb, cc)

    this_line_indents_wrongly
})
```

**Comment by lionel-** [comment]
Also fixed #412 in the process.

The fix doesn't work well with the cascading style, but I think we should just remove it. Probably no one uses that style anymore? This is how ESS historically indented pipelines, but it's no longer our default and I never see this style in the wild.

**Comment by vspinu** [comment]
@lionel-, sure AFAIC cascading can go. I actually forgot that we even have it and how to configure it :)
